src/mlpack/core/data/types.hpp

```diff
@@ -41,8 +38,10 @@ enum struct FileType
               // classes
   HDF5Binary, // HDF5: open binary format, not specific to Armadillo, which can
               // store arbitrary data
-  CoordASCII  // simple co-ordinate format for sparse matrices (indices start at
+  CoordASCII, // simple co-ordinate format for sparse matrices (indices start at
               // zero)
+  ArffASCII   // ARFF data format, with a header specifying information about
```

Comment:
I think this should be `ARFFASCII` to match `CSVASCII`, since `ARFF` is an acronym (and so is CSV).  Let me know what you think.